src/index.jsx

```diff
@@ -1,11 +1,26 @@
-import React from 'react';
-import { render } from 'react-dom';
-import './style.css';
+import React from "react";
+import { render } from "react-dom";
+import Header from "./components/Header";
+import Footer from "./components/Footer";
+import Dresses from "./components/Dresses";
+import Accessories from "./components/Accessories";
+import Shoes from "./components/Shoes";
+import "./style.css";
 
 const App = () => (
-  <>
-    <p>Tady asi něco bude</p>
-  </>
+  <div className="container">
+    <Header title="Móda" subtitle="Stylové oblečení a doplňky pro každý den" />
+
+    <Dresses titleD="Šaty" />
+    <Shoes titleS="Boty" />
+    <Accessories titleA="Doplňky" />
```

Comment:
(pokud to není záměr) nemusíš pojmenovávat props každé komponenty unikátně. Všechny 3 můžou mít jen "title", protože pro každou to je její vlastní proměnná, nemíchají se. Tj klidně jde 
```
    <Dresses titleD="Šaty" />
    <Shoes titleS="Boty" />
    <Accessories titleA="Doplňky" />
```
a pak
```
const Shoes = ({ title }) => {
  return (
    <div className="shoes">
      <h2>{title}</h2>
      <div className="products">
```
a
```
const Dresses = ({ title }) => {
  return (
    <div className="dresses">
      <h2>{title}</h2>
```
a nebude se to míchat